src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Boolean.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Boolean(arg:s|n)
+    // Corresponding Excel and DAX function: Boolean
+    internal sealed class BooleanFunction : BuiltinFunction
+    {
+        public const string BooleanInvariantFunctionName = "Boolean";
+
+        public override bool RequiresErrorContext => true;
+
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
```

Comment:
Why not allow param coercion?

Reply:
See the above question: What would Boolean("1") return?

Reply:
Added negative tests to prevent this flag from changing.